test/hooks/useSelector.spec.js

```diff
@@ -155,15 +155,16 @@ describe('React', () => {
           expect(renderedItems.length).toBe(1)
         })
 
-        it('re-uses the selector if deps do not change', () => {
+        it('re-uses the selector if it does not change', () => {
```

Comment:
Actually, perhaps we should get rid of this test now...

Reply:
yeah, this test was specifically for `deps`, so I would remove it